config/checkstyle/checkstyle-suppressions.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+  "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+  "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+<suppressions>
+  <!-- clash with google java format -->
+  <suppress files=".*\.java" checks="SingleLineJavadoc"/>
+  <suppress files=".*\.java" checks="Indentation"/>
+  <suppress files=".*\.java" checks="RightCurly"/>
+  <suppress files=".*\.java" checks="LineLength"/>
+
+  <!-- temporary suppressions as we work through them -->
```

Comment:
So hopefully we will eventually fix these violations?

Reply:
They should be fixed, but didn't want to complicate this PR. I will work on them next.